util.cpp

```diff
@@ -17,6 +17,13 @@
 
 #include "util.h"
 #include "position.h"
+#include <time.h>
+
+double getProgramTime() {
```

Comment:
You can use Clock::getRealMillis() (static method)

Reply:
It's not accurate enough. Would it be OK if I added Clock::getRealMicros() ?